common/core/web/input-processor/src/text/inputProcessor.ts

```diff
@@ -88,10 +88,10 @@ namespace com.keyman.text {
 
         // Can the suggestion UI revert a recent suggestion?  If so, do that and swallow the backspace.
         if((keyEvent.kName == "K_BKSP" || keyEvent.Lcode == Codes.keyCodes["K_BKSP"]) && this.languageProcessor.tryRevertSuggestion()) {
-          return;
+          return new RuleBehavior();
           // Can the suggestion UI accept an existing suggestion?  If so, do that and swallow the space character.
         } else if((keyEvent.kName == "K_SPACE" || keyEvent.Lcode == Codes.keyCodes["K_SPACE"]) && this.languageProcessor.tryAcceptSuggestion('space')) {
-          return;
+          return new RuleBehavior();
```

Comment:
Note:  these two lines are never actually evaluated.

These blocks are intended as a 'switch' should we ever enable _automatic_ acceptance or reversion of suggestions.  That's currently 100% disabled.

In either case, this is the best pathway forward, as it provides a "truthy" value that will be interpreted correctly by any of the method's callers.